requirements.txt

```diff
@@ -1,3 +1,3 @@
-black==23.3.0
-Pillow==10.0.0
-opencv-python==4.8.0.74
+black
```

Comment:
This shouldn't be in requirements.txt

https://github.com/frappe/frappe/blob/develop/pyproject.toml#L101